app/javascript/components/miq-alert-set-form/miq-alert-set-form.schema.js

```diff
@@ -51,13 +51,16 @@ function createSchema(fieldss, recordId, emsId, mode, loadSchema, alertState, al
               name: 'alert_profile_alerts',
               key: `alerts-${emsId}`,
               label: __('Alert Selection'),
-              leftTitle: emsId ? changeTitle(emsId, mode) : 'Avaliable Alerts',
+              leftTitle: emsId ? changeTitle(emsId, mode) : __('Avaliable Alerts:'),
```

Comment:
haha i just noticed the original had a typo anyway

```suggestion
              leftTitle: emsId ? changeTitle(emsId, mode) : __('Available Alerts:'),
```